object_template.go

```diff
@@ -0,0 +1,86 @@
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"runtime"
+	"unsafe"
+)
+
+// PropertyAttribute are the attributes for a property
+type PropertyAttribute uint8
+
+const (
+	// None.
+	None PropertyAttribute = 0
```

Comment:
That would be incorrect, because that would mean `ReadOnly` would result in `1 << 1` and it needs to be `1 << 0`
As per the doc:
> enum | PropertyAttribute { None = 0,  ReadOnly = 1 << 0,  DontEnum = 1 << 1,  DontDelete = 1 << 2  }
https://v8.github.io/api/head/namespacev8.html

Reply:
Ah sorry, you're right there.

This does need the type on the iota line else these end up as untyped constants.

Reply:
good spot. Without it would end up as an int.